.vsts-ci/templates/ci-general.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  # Setup
+  - pwsh: |
+      git clone https://github.com/PowerShell/PowerShellEditorServices.git ../PowerShellEditorServices
```

Comment:
This really makes me think PSES should be a submodule. Otherwise you will need to handle the case of it checking out the legacy branch for legacy builds.
Because submodules are evil, I think PSES should just be merged into the vscode repo.

Reply:
It's nice living in its own repo.

There are other extensions that depend on it - IntelliJ, Visual Studio, Vim, etc.